cmd/frontend/graphqlbackend/textsearch.go

```diff
@@ -46,6 +47,8 @@ var (
 			},
 		},
 	}
+
+	lineNumberRe = regexp.MustCompile(":(\\d*)$")
```

Comment:
```suggestion
	lineNumberRe = regexp.MustCompile(":(\\d+)$")
```

I think we only want to do this if we match at least one digit.

Reply:
My reasoning here was that a search for `filename.ext:` should just be treated as a search for `filename.ext` in the same way that `foo.com?` is roughly equivalent to `foo.com`. When the capturing group is an empty string, then `strconv.ParseInt` fails and no synthetic match is created. Is all of this too much assumption/magic?

Reply:
I think we should only match here if there is a digit